packages/chips/chip-set/index.js

```diff
@@ -0,0 +1,67 @@
+import React, {Component} from 'react';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import {MDCChipSetFoundation} from '@material/chips';
+
+import Chip from '../chip';
+
+export default class ChipSet extends Component {
+
+  foundation_ = null;
+
+  constructor(props) {
+    super(props);
+    this.maxId = 0;
+    this.state = {
+      chips: props.labels.map((label) => {
```

Comment:
It is an anti-pattern to set `props` that will never change to `state`. Consider using props directly in the `render` method.